src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -165,6 +179,14 @@ public function tolerates(array $deprecationGroups)
         return true;
     }
 
+    /**
+     * @return bool
+     */
+    public function isIgnoredDeprecation(Deprecation $deprecation)
+    {
+        return (bool) preg_filter($this->ignoreDeprecationPatterns, '$0', $deprecation->getMessage());
```

Comment:
The patterns are user-provided patterns, which means it's important to do error handling with `preg_last_error()` and `preg_last_error_msg` IMO

Reply:
done